From c99cf18c142df2172649efeaa9922616c02847d7 Mon Sep 17 00:00:00 2001
From: xxh <xxh@rock-chips.com>
Date: Fri, 22 Aug 2014 10:49:57 +0800
Subject: [PATCH]      1.ADD 3126 DTS for ESP8089 WIFI      2.3126 ESP8089 WIFI
 is controled by PMIC      3.remove regulator-always-on property in file
 rk818.dtsi to contril WIFI power

---
 arch/arm/boot/dts/rk3126-sdk.dts | 17 ++++++++++++++++-
 1 file changed, 16 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3126-sdk.dts b/arch/arm/boot/dts/rk3126-sdk.dts
index 0a9c90dc473b..52b69333bfb0 100755
--- a/arch/arm/boot/dts/rk3126-sdk.dts
+++ b/arch/arm/boot/dts/rk3126-sdk.dts
@@ -19,7 +19,22 @@
 		rockchip,remote_wakeup;
 		rockchip,usb_irq_wakeup;
 		};
-
+        
+        wireless-wlan {		
+	       compatible = "wlan-platdata";
+	       wifi_chip_type = "esp8089";
+	       sdio_vref = <0>; //1800mv or 3300mv
+	       
+	       power_ctrl_by_pmu;
+	       power_pmu_regulator = "rk818_ldo8";
+	       power_pmu_enable_level = <1>; //1->HIGH, 0->LOW
+	       
+	       //vref_ctrl_enable;
+	       //vref_pmu_regulator = "rk818_ldo8";
+	       //vref_pmu_enable_level = <1>; //1->HIGH, 0->LOW
+	       
+	       satus = "okay";	
+	 };	
 };
 
 &rk_screen {
-- 
2.35.3

